typescript/cli/src/deploy/warp.ts

```diff
@@ -136,16 +136,23 @@ export async function runWarpRouteDeploy({
 
   await runDeployPlanStep(deploymentParams);
 
+  const ethereumChains = chains.filter(
```

Comment:
maybe leave a comment that you're filtering here because the functions below throw instead of just skipping if they encounter an ethereum chain